src/components/notifications/content/TwitterShare.tsx

```diff
@@ -117,11 +119,29 @@ const getRemixCosignText = async (
     link
   }
 }
+
 export const getRewardsText = (notification: ChallengeReward) => ({
   text: `I earned $AUDIO for completing challenges on @AudiusProject #AudioRewards`,
   link: null
 })
 
+const tierInfoMap: Record<BadgeTier, { label: string; icon: string }> = {
+  none: { label: 'None', icon: '' },
+  bronze: { label: 'Bronze', icon: '🥉' },
+  silver: { label: 'Silver', icon: '🥈' },
+  gold: { label: 'Gold', icon: '🥇' },
+  platinum: { label: 'Platinum', icon: '🥇' }
```

Comment:
same comment as in web client lol, wondering if this is the correct icon for platinum...